duckduckgo/Android
# Fireproof websites empty state and Feature pixels

Task/Issue URL: https://app.asana.com/0/0/1170962402828809/f
Tech Design URL: 
CC: 

## **Description**:
This PR includes two different things:
- Empty state for fireproof websites screen
- Feature pixels

**Implement empty state for fireproof websites screen** 
| Light | Dark |
| ------ | ----- | 
<image src="https://user-images.githubusercontent.com/4747865/81385739-28e9b600-9114-11ea-8a5f-a5fe2576af03.png" width="240px"/>|<image src="https://user-images.githubusercontent.com/4747865/81385781-399a2c00-9114-11ea-8e45-94307f2c4625.png" width="240px"/>

**Implement feature pixels**
- User clicks on "fireproof a website"
- User undo "fireproof website" action (confirmation snackbar after fireproffing a website)
- User removed a website from "fireproof websites"

## **Steps to test this PR**:
**Test 1: fireproof websites empty state**
1. Fresh install
1. Access settings -> fireproof websites
1. Ensure list is empty and "No websites fireproofed yet"
1. Go back to browser
1. Fireproof any website
1. Navigate to settings -> fireproof websites again
1. Ensure list is not empty
1. Remove all websites from the list
1. Ensure  "No websites fireproofed yet" appeared again

**Test 2: fireproof website pixel**
1. Go back to browser
1. Fireproof any website
1. Ensure pixel sent: `m_fw_a`

**Test 3: Undo fireproof website pixel**
1. Go back to browser
1. Fireproof any website
1. Click on Undo snackbar action
1. Ensure pixel sent: `m_fw_u`

**Test 4: Remove fireproof website pixel**
1. Go back to browser
1. Fireproof any website
1. Navigate to settings -> fireproof websites again
1. Remove a website from the list
1. Ensure pixel sent: `m_fw_d`

**Test 5: Description on top**
1. Ensure description always appears on top of empty state or list of fireproof websites

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
